src/dotnet-svcutil/lib/src/Bootstrapper/SvcutilBootstrapper.cs

```diff
@@ -123,22 +123,17 @@ internal async Task GenerateProjectAsync(bool keepBootstrapperDir, ILogger logge
                 this.MSBuildProj = await MSBuildProj.DotNetNewAsync(projectFullPath, logger, cancellationToken).ConfigureAwait(false);
                 this.MSBuildProj.AddDependency(svcutilPkgRef, true);
 
-                // Comment out code below for reasons: 1. it never used for .net core later than V2.1 since when the approach is always use TF from the generated project.
-                // 2. with below code applied when target framework is netstandard2.0 client machine require netcoreapp2.0 (obsolete) for bootstrapper to work
-                // 3. keep it here for future reference in case when we need definite bootstrapper TF version
-
-                // NOTE: If post v2.0 NetStandard ships a different version from NetCore the table below needs to be updated!
-                //var targetFramework = frameworkInfo.FullName;
-                //if (isSupportedTFM && frameworkInfo.IsKnownDnx)
-                //{
-                //    if (frameworkInfo.Name == FrameworkInfo.Netstandard)
-                //    {
-                //        targetFramework = FrameworkInfo.Netcoreapp + TargetFrameworkHelper.NetStandardToNetCoreVersionMap[frameworkInfo.Version];
-                //    }
-                //    this.MSBuildProj.TargetFramework = targetFramework;
-                //}
+                var targetFramework = frameworkInfo.FullName;
+                if (isSupportedTFM && frameworkInfo.Name != FrameworkInfo.Netstandard && frameworkInfo.Version.CompareTo(new Version(6, 0)) >= 0)
```

Comment:
Why rule out Net standard and 6.0 here?

Reply:
The bootstrapper project is an executable project, so NetStandard is excluded here.
.NET target lower than 6.0 are ruled out to be consistent with previous behavior, back to .net5.0 and netcoreapp3.1, bootstrapper project uses the default target framework from the generated project.